### PR TITLE
Add border attribute for Body, Column and Section components.

### PR DESCRIPTION
**Pull request by hikouki**
:sushi: Add border attribute for Body, Column and Section components. #92

### CODE REVIEW

File: src/components/Column.js [hunk 1]
```diff
@@ -31,7 +31,12 @@ class Column extends Component {
         fontSize: "13",
         textAlign: "left",
         width: "100%",
-        minWidth: mjAttribute('width')
+        minWidth: mjAttribute('width'),
+        border: mjAttribute('border'),
```
Comment (iRyusa) [inline src/components/Column.js]: Is this ok on outlook ? I don't think they support `border` on `div`

File: src/components/Body.js [hunk 2]
```diff
@@ -22,7 +22,12 @@ class Body extends Component {
     return {
       div: {
         backgroundColor: mjAttribute('background-color'),
-        fontSize: mjAttribute('font-size')
+        fontSize: mjAttribute('font-size'),
+        border: mjAttribute('border'),
```
Comment (iRyusa) [inline src/components/Body.js]: Same comment here

**Comment by iRyusa** [comment]
Thanks for your PR ! :+1: 
Can you check that's border is well supported on outlook client ? So we can merge that and update the doc !

**Comment by hikouki** [comment]
Thank you for your reply ! :sushi: 

I only revised that a little.

It seems that border attribute no problem for outlook.

outlook

**Comment by iRyusa** [comment]
Yes looks good I'll check on outlook 2010/11/12/13 and then merge it ! 
Awesome work :+1:

**Comment by hikouki** [comment]
@iRyusa 
Sorry.. I made misunderstanding about outlook client.
I'll check it again on outlook client if it's possible.
:cat:  Thank you anyway!

**Comment by iRyusa** [comment]
Outlook.com is one part of the supported client too, it's good :D 

If you want try something such as litmus or emailonacid !

**Comment by hikouki** [comment]
Those are great services :+1:
Thank you for telling me!

I tried litmus and it seems broken layout for outlook 2010. :crying_cat_face: 
I'll leave it to your judgement if it's merged or not.

outlook2013

**Comment by JeroenSormani** [comment]
Love to see this merged. 
Just my 2 cents, but Outlook 2010 is 5,5 years old, and official support for it stopped last year. IMO that shouldn't be a factor :smile:

**Comment by iRyusa** [comment]
Will try to merge it after `1.3` release, sorry for the delay @hikouki 
cc @JeroenSormani

**Comment by cinghaman** [comment]
Hi, is this part of production code now, cant seem to add border to section the way it shows in example above
thanks

**Comment by JordyPouw** [comment]
@iRyusa Is this already part of version 2.3.2? Border don't seem to work for me.

**Comment by iRyusa** [comment]
Some will be available on next big version via #321 but nothing on 2.3 atm

**Comment by JordyPouw** [comment]
@iRyusa Thanks for the reply!

**Comment by iRyusa** [comment]
Hi there

We've added Border on `mj-column` & `mj-section` in MJML3 ! Both are now ok in every clients and doesn't fail on Outlook. 

So I'm closing this PR, thanks for your contribution !

**Pull request closed without merging**